src/bun.js/node/util/format.zig

```diff
@@ -0,0 +1,406 @@
+const std = @import("std");
+//const bun = @import("root").bun;
+const bun = @import("../../../bun.zig");
+const string = bun.string;
+const strings = bun.strings;
+const String = bun.String;
+//const JSC = bun.JSC;
+const JSC = @import("../../../bindgen.zig").JavaScriptCore;
+const JSValue = JSC.JSValue;
+const JSGlobalObject = JSC.JSGlobalObject;
+const ConsoleObject = @import("../../ConsoleObject.zig");
+const ZigString = JSC.ZigString;
+const MutableString = @import("../../../string_mutable.zig").MutableString;
```

Comment:
```suggestion
const bun = @import("root").bun;
const string = bun.string;
const strings = bun.strings;
const String = bun.String;
const JSC = bun.JSC;
const JSValue = JSC.JSValue;
const JSGlobalObject = JSC.JSGlobalObject;
const ConsoleObject = JSC.ConsoleObject;
const ZigString = JSC.ZigString;
const MutableString = bun.MutableString;
```
this should fix the build